app/Http/Controllers/Api/OrderController.php

```diff
@@ -35,7 +73,76 @@ public function create()
      */
     public function store(Request $request)
     {
-        $request = $request;
+        DB::beginTransaction();
+        try {
+            $order = $this->order->create(
+                [
+                    'user_id' => $request->input('user_id'),
+                    'trans_at' => $request->input('trans_at'),
+                    'custom_address' => $request->address_ship,
+                    'status' => 1
+                ]
+            );
+            if ($request->type == 'App\Food') {
+                $itemtable = new Food();
+//                dd(date("d/m/Y", strtotime($request->trans_at)));
```

Comment:
Please remove debug code. Use comment to make clear code, don't use to remove debug code.